### ISSUE TITLE
Nested RadSideDrawer have strange offset on both main content and drawer content

### ISSUE THREAD

**Issue by gogoout** [NativeScript/nativescript-ui-feedback#956]
> Please take a minute to read our [NativeScript Code of Conduct](https://github.com/NativeScript/codeofconduct) before proceeding with posting issues or discussing. The purpose of this guide is to make communication and cooperation within our forums a pleasure for you and the other members.

### Did you verify this is a real problem by searching the [NativeScript Forum](https://discourse.nativescript.org/)?
Yes

### Tell us about the problem
Nested RadSideDrawer have strange offset on both main content and drawer content.

### Which platform(s) does your issue occur on?
iOS

### Please provide the following version numbers that your issue occurs with:
- **Progress NativeScript UI version:** "nativescript-ui-sidedrawer": "5.0.0"
- **Cross-platform modules:** "tns-core-modules": "5.0.1" (also tried `^5.0.5-rc-2018-11-26-163340-02`)
- **Runtime(s):** `tns-ios 5.0.0`

### Please tell us how to recreate the issue in as much detail as possible.
1. Start the application
2. There are offset on top of the RadSideDrawer main content
3. Click Open Drawer see the offset on RadSideDrawer drawer content
<img width="239" alt="screen shot 2018-12-02 at 13 45 56" src="https://user-images.githubusercontent.com/7570175/49334354-a0223d00-f638-11e8-9aef-94aa6e00447d.png">

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.
https://play.nativescript.org/?template=play-ng&id=Vxph50

**Comment by gogoout** [comment]
Hi there,
Please note this bug is not the same one I reported last day https://github.com/NativeScript/nativescript-angular/issues/1638  as this one is nested.
I'm guessing the issue is due to in `v5.0.0`, the nested RadSideDrawer is brutally set to `iosOverflowSafeArea="false"` which can't be changed by user.

Additionally, I find the  behaviour is quite confusing as according to the [doc](https://docs.nativescript.org/angular/ui/layouts/layouts).  

> The default behavior is that certain container View components (these that can have children) overflow the safe area and are laid out to the edges of the screen.

The RadSideDrawer is nothing but a container, so why is it different from the others?

**Comment by BasyaLipman** [comment]
I think I am experiencing same issue @gogoout . Nesting the RadSideDrawer within 
`<GridLayout rows="*">` helped for the gap at the bottom, and caused the side drawer to overflow and stretch to cover the bottom of the screen. However then I was left with a space ABOVE the side drawer.

Here's how it looks with the gap at the bottom of the screen (no `<GridLayout rows="*">` wrapper):
<img width="461" alt="Screen Shot 2019-04-09 at 10 29 40 PM" src="https://user-images.githubusercontent.com/1658862/55847721-5f6f8e80-5b18-11e9-9740-91822be4fb93.png">

Here's how it looks with the gap on top (with `<GridLayout rows="*">` wrapper):
<img width="445" alt="Screen Shot 2019-04-09 at 10 41 16 PM" src="https://user-images.githubusercontent.com/1658862/55847771-9d6cb280-5b18-11e9-9126-3ef1129a43c9.png">

Here's my template code:
```
<GridLayout id="app" rows="*" class="main-background">
	<RadSideDrawer drawerContentSize="350" row="0" col="0" rowSpan="2" [drawerLocation]="currentLocation">
		<GridLayout tkDrawerContent rows="*">
			<core-more-drawer *ngIf="currentDrawer === 'more'" (close)="closeDrawer($event)"></core-more-drawer>
			<core-transfer-drawer *ngIf="currentDrawer === 'transfer'" (close)="closeTransferDrawer($event)">
			</core-transfer-drawer>
		</GridLayout>

		<GridLayout tkMainContent #container [@routerTransition]="selectedTab" rows="*, 75" columns="*">
			<page-router-outlet row="0"></page-router-outlet>

			<CardView [visibility]="isDisplayBottomNav ? 'visible' : 'collapse'" elevation="20" shadowOffsetHeight="0"
				shadowRadius="20" shadowOpacity="0.35" row="1">
				<core-bottom-nav (openDrawer)="openDrawer($event)" (openTransferDrawer)="openTransferDrawer($event)">
				</core-bottom-nav>
			</CardView>
		</GridLayout>
	</RadSideDrawer>
</GridLayout>
```

@NickIliev do you know a workaround for this issue in the meantime?

Thanks!

**Comment by BasyaLipman** [comment]
@tsonevn do you have insight on this issue?

**Comment by tsonevn** [comment]
Hi @BasyaLipman,
The issue is still under review and I would suggest keeping track on it for further info.
Something that you can try is to define the RadSideDrawer as a root component, which should stretch it on the whole screen surface.  Check out our template [here](https://github.com/NativeScript/nativescript-app-templates/blob/master/packages/template-drawer-navigation-ng/src/app/app.component.html).

**Comment by tylerablake** [comment]
Hi @tsonevn, any update on this?

**Comment by tsonevn** [comment]
Hi @tylerablake 
As far as I know, the status of the issue is still the same. Meanwhile, I would suggest checking whether there is an improvement with the latest nativescript-ui-sidedrawer version `6.0.0`.

**Comment by tylerablake** [comment]
@tsonevn Unfortunately I am running version `6.0.0` already :(

**Comment by tgpetrov** [comment]
Released with [nativescript-ui-sidedrawer@7.0.0](https://github.com/NativeScript/nativescript-ui-feedback/blob/master/releases/sidedrawer.md#700-2019-july-17)
